kamailio/message-role.cfg

```diff
@@ -1,5 +1,52 @@
 modparam("htable", "htable", "msg=>size=32;autoexpire=60;")
 
+route[HANDLE_MESSAGE]
+{
+    if (!is_method("MESSAGE")) return;
+
+    xlog("L_INFO", "$ci|log|MESSAGE from $fu to $tu\n");
+    if (isflagset(FLAG_INTERNALLY_SOURCED) || src_ip == myself) {
+        xlog("L_INFO", "$ci| routing MESSAGE to external from $fu to $tu\n");
```

Comment:
@lazedo log line has the wrong prefix, "$ci|log|XXX" :)

Reply:
will fix